nvtabular/inference/triton/model_hugectr.py

```diff
@@ -51,6 +51,9 @@ def initialize(self, args):
         self.workflow = nvtabular.Workflow.load(workflow_path)
         self.model_config = json.loads(args["model_config"])
         self.column_types = get_column_types(workflow_path)
+        self.slot_sizes = get_slot_sizes(workflow_path)
+        self.slot_sizes.insert(0, 0)
+        self.slot_sizes.pop()
```

Comment:
Does this work? The slot sizes file is written out as a dictionary afaict on line 138 of ```__init__.py``` , and this code seems to be expecting a list

Reply:
Forgot to push some of the updates before turning the title to "REVIEW". Just pushed them.